app/models.js

```diff
@@ -141,13 +141,31 @@ class Deployment extends K8sModel {
     return 'deployments';
   }
 
+  static restart(label) {
+    return Pod.delete_all({ labelSelector: `app=${label}` });
+  }
+
   get_pods() {
     return Pod.list({ labelSelector: `app=${this.data.metadata.labels.app}` });
   }
 
   restart() {
     return Pod.delete_all({ labelSelector: `app=${this.data.metadata.labels.app}` });
   }
+
+  get_status() {
+    for (let i in this.status.conditions) {
```

Comment:
Isn't this equivalent to:

```javascript
for (let condition in this.status.conditions) {
  // ...
}
```

Reply:
No, `for .. in` iterates through the indices (0, 1, 2, ...).
Perhaps you meant `for .. of`, which I didn't realise works with arrays! I will change it.

Reply:
Yep, damn you JS :P